src/index.js

```diff
@@ -0,0 +1,5 @@
+var object = require('./object')
+
+module.exports = {
```

Comment:
I am not sure we can follow which naming conventions, but will will discuss.

Reply:
My suggestion will be to use camelCase.